inst/rmarkdown/templates/mdpi/skeleton/skeleton.Rmd

```diff
@@ -128,45 +160,83 @@ Numbered lists can be added as follows:
+%%% \tablesize{} %% You can specify the fontsize here, e.g., \tablesize{\footnotesize}. If commented out \small will be used.
 \begin{tabular}{ccc}
 \toprule
 \textbf{Title 1}	& \textbf{Title 2}	& \textbf{Title 3}\\
 \midrule
-entry 1		& data			& data\\
-entry 2		& data			& data\\
+Entry 1		& Data			& Data\\
+Entry 2		& Data			& Data\\
 \bottomrule
 \end{tabular}
-\end{table}
+\end{specialtable}
 
-This is an example of an equation:
+Note that using default Markdown tables will put these in a normal table environment which may cause issues with numbering. To pass the 'specialtable' environment you can use knitr::kable (and the following arguments get you approximately the same style as the journal's.
 
+```{r knittable, echo=FALSE}
+knitr::kable(mtcars[1:5, 1:3], format = "latex", table.envir = "specialtable", 
+             booktabs = TRUE, 
+             caption = "This is my caption", position = "H", 
+             align='ccc', centering = FALSE)
+```
+
+Note also that longtable will not work in the two column format, see [here](https://tex.stackexchange.com/questions/161431/how-to-solve-longtable-is-not-in-1-column-mode-error) for some workarounds, which you can pass to the preamble.tex (see [here](https://bookdown.org/yihui/rmarkdown-cookbook/latex-preamble.html) for more details). 
+
+## Formatting of Mathematical Components
+  
+This is the example 1 of equation:
 \begin{equation}
-\mathbb{S}
+a = 1,
 \end{equation}
+the text following an equation need not be a new paragraph. Please punctuate equations as regular text.
 <!-- If the documentclass option "submit" is chosen, please insert a blank line before and after any math environment (equation and eqnarray environments). This ensures correct linenumbering. The blank line should be removed when the documentclass option is changed to "accept" because the text following an equation should not be a new paragraph. -->
 
-<!-- Please punctuate equations as regular text. Theorem-type environments (including propositions, lemmas, corollaries etc.) can be formatted as follows: -->
-Example of a theorem:
+This is the example 2 of equation:
+
+```{=latex}
+\end{paracol}
+\nointerlineskip
+\begin{equation}
+a = b + c + d + e + f + g + h + i + j + k + l + m + n + o + p + q + r + s + t + u + v + w + x + y + z
+\end{equation}
+```
+
+
+<!-- % Example of a figure that spans the whole page width (the commands \widefigure and \begin{paracol}{2}, \linenumbers, and\switchcolumn need to be present). The same concept works for tables, too. -->
+```{=latex}
+\begin{figure}[H]	
+\widefigure
+\includegraphics[width=15 cm]{Definitions/logo-mdpi}
+\caption{This is a wide figure.\label{fig2}}
+\end{figure}  
+\begin{paracol}{2}
+\linenumbers
+\switchcolumn
+```
+
+Theorem-type environments (including propositions, lemmas, corollaries etc.) can be formatted as follows:
+
 \begin{Theorem}
 Example text of a theorem.
 \end{Theorem}
```

Comment:
FYI interest, with recent **rmarkdown**, this should now work

````
::: {.Theorem latex=true}
Example text of a theorem
:::
````

https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html